09-PowershellModule/Readme.md

```diff
@@ -0,0 +1,133 @@
+
+
+
+## List of commonly used parameters
+
+| Parameter (alias) |
+|-------------------|
+|SubscriptionId (sid)|
+|TenantId (tid)|
+|FilterTags (ftg)|
+|ResourceTypeNames (rtn)|
+|ExcludeResourceTypeNames (ertn)|
+|ControlIds (cids)|
+|ExcludeControlIds (ecids)|
+
+----------------------------------------------------------
+## AzTS: Security scanner for Azure
+
+>  **Prerequisites:**
+> For all commands in this feature it is assumed that you have:
+> 1. Logged in to your Azure account using Connect-AzAccount from a PS/ISE console.
+>```PowerShell
+>   Connect-AzAccount 
+>```
+> 2. Selected a subscription using Set-AzContext.
+>```PowerShell
+>	Set-AzContext -SubscriptionId <SubscriptionId> 
+>```
+
+### Overview
+ 
+Security Scanner for Azure scans all the prominent features in Azure (e.g., Web Apps, Storage, SQL DB, Key Vault, etc.) by
+executing the security controls and will validate their status as 'Success' or 'Failure' based on the security guidance.
+
+The following aspects of security are also checked:
+1. Access control configuration - identity and access management related issues in the subscription.
+2. Microsoft Defender for Cloud configuration - configuration of MDC (security point of contact, various MDC policy settings, etc.)
+
+### Scan the security of Azure
+
+The security check script for Azure can be run using the command below after replacing `<SubscriptionId`> 
+ with your subscriptionId and  `<TenantId`> with your tenantId of subscription.
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId> -TenantId <TenantId>
+```
+The parameters used are:
+- SubscriptionId – Subscription ID is the identifier of your Azure subscription
+- TenantId - Tenant ID is the identifier for your Azure tenant 
+
+You need to have at least **Reader** role at the subscription scope to run this command. 
+
+### Security scanner for azure - How to fix findings?
+
+AzTS cmdlet generate outputs which are organized as under: 
+- Summary information of the control evaluation (pass/fail) status in a CSV file
+
+To address findings, you should do the following:
+1. See the summary of control evaluation first in the CSV file. (Open the CSV in XLS. Use "Format as Table", "Hide Columns", "Filter", etc.)
+2. Review controls that are marked as "Failed" or "Verify"
+3. The 'Remediation Steps' column for each control in the XLS will tell you the command/steps needed to resolve the issue.
+
+
+### Target specific controls during security scan for azure
+
+The security scanner for Azure supports multiple parameters as specified below:
+- SubscriptionId – Subscription ID is the identifier of your Azure subscription 
+- TenantId - Tenant ID is the identifier of your Azure subscription
+- FilterTags  - Comma separated tags to filter the security controls. e.g.: RBAC, Automated, etc.
+- ControlIds  - Comma separated control ids to filter the security controls. e.g.:Azure_Storage_AuthN_Dont_Allow_Anonymous,"Azure_APIManagement_DP_Use_HTTPS_URL_Scheme, etc.
```

Comment:
Azure_Storage_AuthN_Dont_Allow_Anonymous,"Azure_APIManagement_DP_Use_HTTPS_URL_Scheme, etc 

Double quote before Azure_APIManagement should be removed